.github/setup/action.yaml

```diff
@@ -21,7 +21,7 @@ runs:
 
     - name: Determine poetry version
       shell: bash
-      run: echo "::set-output name=VERSION::$(poetry --version)"
+      run: echo "{VERSION}=$(poetry --version)"
```

Comment:
See https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/  .  We were getting warnings, and this gets rid of them